node/service/src/lib.rs

```diff
@@ -420,6 +441,22 @@ impl TransactionConverters {
 	fn moonbase() -> Self {
 		unimplemented!()
 	}
+	pub fn for_chain_spec(chain_spec: &Box<dyn ChainSpec>) -> Self {
+		match chain_spec {
+			spec if spec.is_moonbeam() => Self::moonbeam(),
+			spec if spec.is_moonriver() => Self::moonriver(),
+			spec if spec.is_moonbase() => Self::moonbase(),
+			_ => panic!("invalid chain spec"),
+		}
+	}
```

Comment:
I'm not using this in the PR, but I found it very useful when I first wrote it (it helps avoid big `#[cfg...]` blocks of code). OTOH, I'm not a fan of dead code...

Reply:
So, please remove this unused code

Reply:
Fixed in 7a35f70